src/odemis/driver/test/simcam_test.py

```diff
@@ -303,6 +317,40 @@ def test_focus(self):
         f.result()
         self.assertEqual(self.focus.position.value, pos)
 
+    def test_stage(self):
+
+
+        pos = self.stage.position.value
+
+        d0 = self.camera.data.get()
+        self.assertEqual(d0.shape, (128,128))
+
+        f = self.stage.moveRel({"x": 1e-3}) # 1 mm
+        f.result()
+        logging.debug('pos= %s',pos)
+        self.assertNotEqual(self.stage.position.value, pos)
+        d1 = self.camera.data.get()
+        self.assertTrue(numpy.any(d0 != d1))
+
+
+        f = self.stage.moveRel({"y": 1e-3}) # 1 mm
+        f.result()
+        self.assertNotEqual(self.stage.position.value, pos)
+        self.camera.data.get()
+
+
+        # restore original position
+        f = self.stage.moveAbs(pos)
+        logging.debug('pos= %s',pos)
+        f.result()
+        self.assertEqual(self.stage.position.value, pos)
+        d0n =  self.camera.data.get()
+        numpy.testing.assert_array_almost_equal(d0, d0n)
+
+
+
```

Comment:
No need for all these empty lines. Please put maximum 1 empty line between blocks.